highway/client/cosmos.go

```diff
@@ -32,10 +38,13 @@ func NewCosmos(ctx context.Context, config *config.Config) (*Cosmos, error) {
 
 	// create a new client instance
 	return &Cosmos{
-		accName:  config.CosmosAccountName,
-		address:  account.Address("snr"),
-		instance: cosmos,
-		query:    types.NewQueryClient(cosmos.Context),
+		accName:       config.CosmosAccountName,
```

Comment:
Do we need the session context in this struct? I feel we dont as this is on chain but we might want to keep a reference to the session? Curious on what you think

Reply:
Yes